packages/suite-desktop/src/support/DesktopUpdater/EarlyAccessEnable.tsx

```diff
@@ -33,6 +25,14 @@ const Checkbox = styled(CheckItem)`
     max-width: 100%;
 `;
 
+const StyledModal = styled(Modal)`
+    ${Modal.BottomBar} {
+        > * {
```

Comment:
I didn't want to, because I'd like to discuss whether we want the buttons to fill the available space or to fit their content, and then to unify it across all the modals. Until that I've just preserved current state.